### DISCUSSION TITLE
萌新绳网求助

### DISCUSSION THREAD

**Discussion by YoKiiiiiiiiiiiii** [share121/inter-knot#1595] [其他]
#代理人
刚刚把瓦尔特·杨从隔壁忽悠过来当代理人了，请问它可以让我在空洞无敌嘛
![IMG_4264](https://github.com/user-attachments/assets/0c7505f9-c018-48c5-95a0-78dfd88ea7c6)

**Comment by match-fire** [comment]
根据欧拉数学定理，两个空洞的叠加态会产生白洞，因此方案可行

**Comment by qingtaixu** [comment]
瓦尔特能操控黑大帅，操控个空洞怎么了？
